DFC.Api.Content/StartUp/FunctionStartupExtension.cs

```diff
@@ -43,5 +45,30 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddSingleton<IJsonFormatHelper, JsonFormatHelper>();
 
         }
+
+        private static string GetCustomSettingsPath()
+        {
+            var home = Environment.GetEnvironmentVariable("HOME");
+            string? path;
+            if (home != null)
+            {
+                // We're on Azure
+                path = Path.Combine(home, "site", "wwwroot");
+            }
+            else
+            {
+                // Running locally
+                path = new Uri(Assembly.GetExecutingAssembly().CodeBase!).LocalPath;
+
+                if (!string.IsNullOrEmpty(path))
+                {
+                    path = Path.GetDirectoryName(path);
+                    DirectoryInfo parentDir = Directory.GetParent(path);
+                    path = parentDir.FullName;
+                }
+            }
+
+            return path ?? string.Empty;
```

Comment:
Would it be better to fail fast and throw an exception if we couldn't figure out a path?

Reply:
Yeah you're right :)